.github/workflows/ci.yml

```diff
@@ -1039,3 +1039,82 @@ jobs:
           check_name: ${{ github.job }} with secrets
           conclusion: ${{ job.status }}
           github_token: ${{ secrets.GITHUB_TOKEN }}
+
+  # TODO: Don't always run the Redshift tests; add impact analysis or integrate with above if possible.
+  redshift:
+    name: redshift (${{ matrix.modules }})
+    runs-on: ubuntu-latest
+    if: ${{ env.TRINO_AWS_ACCESS_KEY_ID != '' && env.TRINO_AWS_SECRET_ACCESS_KEY != '' }}
+    env:
+      REDSHIFT_AWS_REGION: us-east-2
+      REDSHIFT_AWS_AVAILABILITY_ZONE: us-east-2a
+      REDSHIFT_NODE_TYPE: dc2.large
+      REDSHIFT_DATABASE_NAME: testdb
+      REDSHIFT_USERNAME: admin
+      # see plugin/trino-redshift/README.md for details on how to setup the Amazon Redshift CI infrastructure
+      REDSHIFT_SUBNET_GROUP_NAME: cluster-subnet-group-trino-ci
+      REDSHIFT_IAM_ROLES: arn:aws:iam::894365193301:role/redshift-ci
+      REDSHIFT_VPC_SECURITY_GROUP_IDS: "[\"sg-055bbbc00509a0345\"]"
+      REDSHIFT_S3_TPCH_TABLES_ROOT: "s3://trino-ci-redshift-test"
+      REDSHIFT_PUBLICLY_ACCESSIBLE: true
+      TRINO_AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
+      TRINO_AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
```

Comment:
The env vars could have no TRINO_ prefix.
Within this test we won't be talking to different accounts.

This will make `aws-actions/configure-aws-credentials` below unnecessary, right?

Reply:
Actually I was using these variables only to avoid running the `redshift` job altogether in case the secrets are not specified.
Removing the `configure-aws-credentials` step.